packages/firestore/lib/index.js

```diff
@@ -33,6 +33,7 @@ import FirestoreCollectionReference from './FirestoreCollectionReference';
 import FirestoreDocumentReference from './FirestoreDocumentReference';
 import FirestorePath from './FirestorePath';
 import FirestoreQuery from './FirestoreQuery';
+export { Filter } from './FirestoreFilter';
```

Comment:
Probably not needed, as it's already in `FirestoreStatics`

Reply:
nice! I've removed the explicit import! Thanks!